src/providers/novita-ai/api.ts

```diff
@@ -0,0 +1,20 @@
+import { ProviderAPIConfig } from '../types';
+
+const NovitaAIApiConfig: ProviderAPIConfig = {
+  getBaseURL: () => 'https://api.novita.ai/v3/openai',
+  headers: ({ providerOptions }) => {
+    return { Authorization: `Bearer ${providerOptions.apiKey}` };
+  },
+  getEndpoint: ({ fn }) => {
+    switch (fn) {
+      case 'complete':
+        return '/v1/completions';
```

Comment:
Hi it's correct, we use the `/v3/openai` prefix, so the complete URL will be /`v3/openai/v1/completion`

Reply:
The final url as mentioned in the documentation is` https://api.novita.ai/v3/openai/completions`
But the above configuration will result in final url being `https://api.novita.ai/v3/openai/v1/completions`

Nothing in the documentation mentions `/v1` being needed

Reply:
hey! I noticed that both `https://api.novita.ai/v3/openai/v1/completions` and `https://api.novita.ai/v3/openai/completions` are behaving similarly. But the novita's official documentation does not mention `https://api.novita.ai/v3/openai/v1/completions` being the end point as mentioned here https://novita.ai/get-started/llm.html#example-with-curl-client. Can you remove the `/v1` so that it is compliant with their official documentation?